<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>page_3</class>
 <widget class="QWidget" name="page_3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>PibotoLt</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pBack_page3">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>330</y>
     <width>150</width>
     <height>150</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>PibotoLt</family>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="Resources.qrc">
     <normaloff>:/new/prefix1/arrows_back_icon.png</normaloff>:/new/prefix1/arrows_back_icon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>90</width>
     <height>90</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Aa삐삐롱</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: 40px;
background-color: rgba(89, 141, 204);
color:white;</string>
   </property>
   <property name="text">
    <string>사용시간예약</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>621</width>
     <height>171</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnminimumwidth="3,7">
    <item row="1" column="1">
     <widget class="QLCDNumber" name="AlramTime">
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="digitCount">
       <number>8</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="currentlabel">
      <property name="font">
       <font>
        <family>Aa삐삐롱</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>current time</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="alramlabel">
      <property name="font">
       <font>
        <family>Aa삐삐롱</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>alram time</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLCDNumber" name="CurrentTime">
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="digitCount">
       <number>8</number>
      </property>
      <property name="segmentStyle">
       <enum>QLCDNumber::Flat</enum>
      </property>
      <property name="value" stdset="0">
       <double>0.000000000000000</double>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>500</width>
     <height>146</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="h_Up">
        <property name="text">
         <string>Hour UP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="h_Text">
        <property name="font">
         <font>
          <family>PibotoLt</family>
          <pointsize>30</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="h_Down">
        <property name="text">
         <string>Hour DOWN</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="m_Up">
        <property name="text">
         <string>Minute UP</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="m_Text">
        <property name="font">
         <font>
          <family>PibotoLt</family>
          <pointsize>30</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_Down">
        <property name="text">
         <string>Minute DOWN</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="add">
        <property name="font">
         <font>
          <family>Aa삐삐롱</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>add</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="remove">
        <property name="font">
         <font>
          <family>Aa삐삐롱</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>remove</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
